custom/icds_reports/ucr/data_sources/dashboard/daily_feeding_forms.json

```diff
@@ -308,9 +307,6 @@
       "primary_key": ["supervisor_id","doc_id","repeat_iteration"]
     },
     "sql_column_indexes": [
-      {
-        "column_ids": ["state_id", "timeend"]
```

Comment:
Wont this change affect the speed of this as timeend is used here: https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/distributed/daily_feeding_forms_child_health.py#L61

Reply:
That's one of the queries that I profiled in the linked PR. This index doesn't affect that query because the query does not filter by state_id. Queries must filter on the leading columns of the index to use the index